### PR TITLE
Create borrowing

### PR DESCRIPTION
**Pull request by BohdanKuzik**

### CODE REVIEW

File: borrowing/serializers.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+from rest_framework import serializers
+from borrowing.models import Borrowing
+
+
+class BorrowingSerializer(serializers.ModelSerializer):
```
Comment (BornToLivee) [inline borrowing/serializers.py:5]: Can you please rename to BorrowingCreateSerializer?

Comment (BohdanKuzik) [reply borrowing/serializers.py:5]: Sure

File: borrowing/urls.py [hunk 2]
```diff
@@ -1,5 +1,10 @@
 from django.urls import path
 
-urlpatterns = []
+from borrowing.views import BorrowingCreateView
+
+
+urlpatterns = [
```
Comment (BornToLivee) [inline borrowing/urls.py:5]: i think on this url we want to have list, create should be on /create, no?

**Review by b3v3kt0r — APPROVED** [review]

**Review by b3v3kt0r — APPROVED** [review]

**Review by BornToLivee — APPROVED** [review]
wish me luck with this conflicts :)

**Pull request merged**